cardano-node/src/Cardano/Node/Configuration/Topology.hs

```diff
@@ -35,98 +36,83 @@ import           Ouroboros.Consensus.Util.Condense (Condense (..))
 
 -- | A data structure bundling together a node identifier and the path to
 -- the topology file.
-data TopologyInfo = TopologyInfo {
-    node         :: NodeId
-  , topologyFile :: FilePath
-  }
+data TopologyInfo = TopologyInfo {node :: NodeId, topologyFile :: FilePath}
```

Comment:
I think this contradicts https://github.com/input-output-hk/cardano-node/blob/master/STYLE.md#data-declarations

```
Format records as follows:

data Person = Person
  { firstName :: !String  -- ^ First name
  , lastName :: !String  -- ^ Last name
  , age :: !Int     -- ^ Age
  } deriving (Eq, Show)
```